webapp/src/main/webapp/i18n/vivo_all.properties

```diff
@@ -810,3 +810,46 @@ role_in_presentation_capitalized=Role in Presentation
 advisee_capitalized_first_name=First Name
 advisee_capitalized_lastname=Last Name
 
+add_orcid_id=Add an iD
```

Comment:
These should also be included in https://github.com/vivo-project/VIVO-languages, no?

Reply:
Created a pull request using the English i18n strings: https://github.com/vivo-project/VIVO-languages/pull/21